packages/app/server/trpc/guards/requireMutation.test.ts

```
@@ -0,0 +1,45 @@
+
+    expect(returned).toBe(result);
+  });
+
+  test("throws TRPCError with code BAD_REQUEST when result is undefined (default)", () => {
+    expect.hasAssertions();
+
+    let error: unknown;
+    try {
+      requireMutation(undefined, Operation.Create, "Entity", "1");
+    } catch (error) {
+      error = error;
```

Biome (2.4.14)

[error] Reassigning a catch parameter is confusing.

(lint/suspicious/noCatchAssign)